src/commands/Command.bus.ts

```diff
@@ -36,13 +36,53 @@ class CommandBus {
 
 const commandBus = new CommandBus();
 
+/**
+ * Command: commands
+ * Usage:
+ * Permissions: All
+ */
 commandBus.addHiddenCommand('commands', new ListCommand);
```

Comment:
Permissions states all, but is added as hidden command

Reply:
the visibility doesn't mean its for admins. The visibility is soley for the list-command to know what commands it should show vs those it shouldn't.